cpplint/cpplint.py

```diff
@@ -52,6 +52,11 @@
 import sys
 import unicodedata
 
+try:
```

Comment:
prefer an explicit version check rather than an exception, something like this:

```
if sys.version_info >= (3,):
  xrange = range  # xrange is only used on for loops so this is fine.
```

Reply:
The Python porting best practice is [_Use feature detection instead of version detection_](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection).